### DISCUSSION TITLE
Retry Policy/Mechanisms for Connections to Redis cache using Lettuce client

### DISCUSSION THREAD

**Discussion by rohanand-MS** [redis/lettuce#2448] [Q&A] [answered]
Hi, I am trying to connect to azure redis cache using Lettuce client. When I am setting up client option for connection, I am not getting any option to set Retry policy for the Connection in case of failures or re-connections attempt. 

Is this functionality not supported on Lettuce yet ? If supported, can you please help with any references or docs ?

**Comment by mp911de** [comment, marked as answer]
Check out `ReconnectDelay` at https://lettuce.io/core/release/reference/#clientresources.configuration-settings

**Comment by rohanand-MS** [comment]
Thanks a lot! It's helpful.
